src/vm/ceeload.cpp

```diff
@@ -12757,6 +12757,7 @@ void Module::LogTokenAccess(mdToken token, SectionFormat format, ULONG flagnum)
         GC_NOTRIGGER;
         MODE_ANY;
         PRECONDITION(g_IBCLogger.InstrEnabled());
+        PRECONDITION(flagnum >= 0);
```

Comment:
You're right, I have undone the changes relevant to flagnum. The error was about tkKind.